packages/closer/components/ProjectView/ProjectView.tsx

```
@@ -0,0 +1,192 @@
+                />
+                <label className="text-sm uppercase font-bold flex gap-1">
+                  {timeFrame ?? (
+                    <>
+                      {start && dayjs(start).format(dateFormat)}
+                      {end &&
+                        Number(duration) > 24 &&
+                        ` - ${dayjs(end).format(dateFormat)}`}
+                      {end &&
+                        Number(duration) <= 24 &&
+                        ` - ${dayjs(end).format('HH:mm')}`}{' '}
+                      {end && end.isBefore(dayjs()) && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)